rosdep/base.yaml

```
@@ -3184,13 +3186,17 @@ libftgl-dev:
   fedora: [ftgl-devel]
   gentoo: [media-libs/ftgl]
   ubuntu: [libftgl-dev]
+libgazebo-dev:
+  ubuntu:
```

We can complete the `libgazebo-dev` entry with the rest of supported distributions.
```suggestion
  debian: [libgazebo-dev]
  gentoo: [=sci-electronics/gazebo-11*]
  ubuntu:
    focal: [libgazebo11-dev]
```
Not sure what is the goal of providing two different entries for versioned/unversioned that currently points to the same value. I see that we are matching the package name which could be valuable but I defer the decision to the people administering this repo.